npm/eslint-plugin-dev/lib/index.js

```diff
@@ -252,12 +255,13 @@ module.exports = {
           },
         },
         {
-          files: '*.ts',
+          files: '*.ts*',
           parser: '@typescript-eslint/parser',
           plugins: [
             '@typescript-eslint',
           ],
           rules: {
+            'no-undef': 'off',
```

Comment:
This shouldn't be turned off. It's a pretty important rule. I know it was turned off for the `server` package, but I don't think that should be the case either. I'd rather have `eslint-disable` comments for exceptions than turning it off for everything.

Reply:
This is being turned off specifically for Typescript files only. It is recommended to disable as Typescript itself does better undefined testing: https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors

Reply:
Ah my bad. I was only looking at the diff, but should have looked at the change in the fuller context.